docs/_data/translations.yml

```diff
@@ -3,11 +3,6 @@
   description: Bootstrap 中文文档
   url: http://v4.bootcss.com/
 
-- name: Chinese
-  code: zh
-  description: Bootstrap 4 中文文档教程
-  url: http://boot4.com/
```

Comment:
Actually after visiting `http://wiki.jikexueyuan.com/project/bootstrap4/` a first time (by accessing this url directly, the redirect on `http://boot4.com/` seems to works...
Seems weird.
In order to reproduce, visit `http://boot4.com/` in Incognito mode.

Anyway it's probably better to provide a link to `http://wiki.jikexueyuan.com/project/bootstrap4/` rather than `http://boot4.com/`